src/main.py

```diff
@@ -34,6 +36,8 @@
 @bot.event
 async def on_ready():
 	log.info(f'Logged in as {bot.user}')
+	ending_note = "Powered by " + str(bot.user)[:-5] + "\nFor command {help.clean_prefix}{help.invoked_with}"
```

Comment:
Bit cleaner, and won't explode if Discord for some reason switches to 5 char tags or something

```suggestion
	ending_note = f'Powered by {bot.user.name}\nFor command {help.clean_prefix}{help.invoked_with}'
```

Reply:
Can't use f String since it uses help.clean_prefix somewhere else, updated it to use split and join instead.

Reply:
and force pushed to use bot.user.name, man I'm bad